<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1826</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="add">
        <property name="text">
         <string>Nou</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refresh">
        <property name="text">
         <string>Elimina</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="import_2">
        <property name="text">
         <string>Importeaza</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="project">
        <property name="text">
         <string>Project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="make_doc">
        <property name="text">
         <string>Calculeaza</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show">
        <property name="text">
         <string>Arata</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_dia">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="smallbutt_l">
      <item>
       <widget class="QPushButton" name="p1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p2">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p3">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p4">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p5">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p6">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p7">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p8">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p9">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p10">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p11">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p12">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p13">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p14">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p15">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p16">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p17">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p18">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p19">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p20">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_butt">
        <property name="text">
         <string>++</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="list">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QTreeView" name="projectlist"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1826</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNou"/>
   <addaction name="actionElimina"/>
   <addaction name="actionImporteaza"/>
   <addaction name="actionProiect"/>
   <addaction name="actionCalculeaza"/>
   <addaction name="actionArata_Proiectele"/>
   <addaction name="actionSalvare_Rapida"/>
   <addaction name="actionUndo"/>
  </widget>
  <action name="actionNou">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Nou</string>
   </property>
   <property name="toolTip">
    <string>Adauga un nou rand.</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionElimina">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
   <property name="toolTip">
    <string>Elimina un rand.</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionImporteaza">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Importeaza</string>
   </property>
   <property name="toolTip">
    <string>Adauga materiale in depozit.</string>
   </property>
  </action>
  <action name="actionProiect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Proiect</string>
   </property>
   <property name="toolTip">
    <string>Scoate materiale in depozit.</string>
   </property>
  </action>
  <action name="actionCalculeaza">
   <property name="text">
    <string>Calculeaza</string>
   </property>
   <property name="toolTip">
    <string>Calculeaza proiectul/importarea in depozit.</string>
   </property>
  </action>
  <action name="actionArata_Proiectele">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Arata Proiectele</string>
   </property>
   <property name="toolTip">
    <string>Arata proiectele salvate.</string>
   </property>
  </action>
  <action name="actionSalvare_Rapida">
   <property name="text">
    <string>Salvare Rapida</string>
   </property>
   <property name="toolTip">
    <string>Salveaza pozitia curenta.</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Anuleaza ultima actiune</string>
   </property>
  </action>
  <action name="actionAdauga_Coloana">
   <property name="text">
    <string>Adauga Coloana</string>
   </property>
   <property name="toolTip">
    <string>Adauga o coloana in lista</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>manage_toolactions(QAction*action)</slot>
 </slots>
</ui>
